baseplate/experiments/providers/simple_experiment.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import hashlib
+import time
+
+from .base import Experiment
+from ..._compat import long, iteritems
+
+
+logger = logging.getLogger(__name__)
+
+
+class SimpleExperiment(Experiment):
+    """Base class for simple experiments. Note that this class shares a lot of
```

Comment:
the stuff about sharing code would be better for a comment since it's irrelvant to someone _using_ baseplate and just reading the docs but is important to someone working inside baseplate.

Reply:
removed